donate.html

```diff
@@ -174,9 +174,88 @@ <h1>Our Story</h1>
       </div>
     </div>
   </div>
+ 
+  <!-- jules -->
+    
+  <!-- donor impact -->
+  <div class="donate-impact">
+    <div class="container">
+      <div class="text-center">
+      <!-- header -->
+        <div class="impact-header">
+          <div class="row">
+            <div class="col-md-12"><h1>Your Donation's Impact</h1></div>
+          </div>
+        </div>
```

Comment:
for single elements like this, its unnecessary imo to add the row/col structure - doing something like 
<div class="impact-header> <h1> ..... </h1> </div> would suffice